go/libkb/env.go

```diff
@@ -922,3 +922,7 @@ func (e *Env) GetMountDir() string {
 func (e *Env) GetAppStartMode() AppStartMode {
 	return e.config.GetAppStartMode()
 }
+
+func (e *Env) GetUpdatePublicKey() string {
```

Comment:
Can you use `env.GetCodeSigningKIDs`? It exists already for this purpose! Note that this will return a KID rather than a raw public key, which is important in case we every decide to change algorithm.